<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="model_1">
    <static>true</static>
    <pose>0 0 0 0 0 0</pose>
    <link name="link">
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>file:///home/izan/.ros/mesh.stl</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>0 1 0 1</ambient>
          <diffuse>0 0 0 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
          <emissive>0 0 0 0</emissive>
        </material>
      </visual>
      <!--<collision name="collision">
        <geometry>
          <mesh>
            <uri>file:///home/izan/.ros/mesh.stl</uri>
          </mesh>
        </geometry>
        <surface>
          <contact>
            <ode/>
          </contact>
          <bounce/>
          <friction>
            <ode/>
          </friction>
        </surface>
      </collision>-->
    
  </model>
</sdf>
